.github/workflows/ci.yml

```diff
@@ -24,19 +24,19 @@ jobs:
           FOUNDRY_PROFILE: ci
           FORK_TESTS: false
 
-  test-invariant:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
+  # test-invariant:
```

Comment:
It doesn't seem worth fixing these as they anyways will be replaced by the new Recon based invariant tests soon.